glossary.adoc

```diff
@@ -0,0 +1,95 @@
+[[glossary]]
+== Glossary
+
+|===
+
+| AIA | Advanced interrupt architecture (AIA) is an architecture for handling
+interrupts.
+
+| ABI | Application binary interface (ABI).
+
+| AP | Application processors (AP)s can support commodity operating systems,
+ hypervisors/VMMs and applications software workloads. The AP subsystem
+ may contain several processing units, on-chip caches, and other controllers
+for interfacing with memory, accelerators, and other fixed-function logic.
+Multiple APs may be used within a logical system.
+
+| Attestation | The process by which a relying party can assess the
+security posture of the confidential workload based on verifying a set of
+HW-rooted cryptographically-protected evidence.
+
+| Confidential Computing | A computing paradigm that protects data in use by
+performing computation in a hardware-based, attested TEE.
+
+| CoVE | Confidential VM extension (CoVE) is the set of RISC-V ABI extensions
+defined in this specification that enables confidential computing on RISC-V
+platforms. In some deployment models, the CoVE ABI leverages the RISC-V ISA
+extensions specified in the RISC-V Supervisor Domains specification.
+CoVE is a Trusted Execution Environment (TEE) ABI for Application Processors.
+A supervisor domain that provides HW-isolation for workload data assets when in
+use (user/supervisor code/data) and provides HW-attestable confidentiality and
+integrity protection against specific attack vectors per a specified
+adversary and threat model.
+
+| Host software | All software elements including type-1 or type-2 HS-mode VMM
+and OS; U-mode user-space VMM tools; ordinary VMs hosted by the VMM that
+emulate devices. The hosting platform is typically a multi-tenant platform
```

Comment:
How do we show platform ---> has one or more SDs ---> host SW runs in the SD in HS-mode and U-mode?

Reply:
this was for the hosting software - the platform with SDs is shown in the intro section

Reply:
I was wondering about the line "The hosting platform is typically a multi-tenant platform"
Should we instead say "The hosting SD hosts multiple mutually distrusting..."

Reply:
good idea - will rephrase